src/application/node/application.ts

```diff
@@ -23,6 +24,7 @@ export interface BackendApplicationContribution {
 export class BackendApplication {
 
     private app: express.Application;
+    @inject(ILogger) private logger: ILogger;
```

Comment:
I mean in general, using ctor or property injection... I can put it into to ctor it just seems like it will become bloated at some point?